docs/DOCKERBUILD.md

```diff
@@ -0,0 +1,23 @@
+# Docker Build :hammer_and_wrench:
```

Comment:
Same, for 100 character wraps.

We also like to space out sections with blank lines, so after the headings and before the code blocks. I find it makes it a nicer text editing experience when there is some room to separate thoughts.